companion/manifest.json

```diff
@@ -0,0 +1,26 @@
+{
+	"id": "kiloview-encoder",
+	"name": "kiloview-encoder",
+	"shortname": "kiloview",
+	"description": "Module to control Kiloview Encoders",
+	"version": "1.0.1",
+	"license": "MIT",
+	"repository": "git+https://github.com/bitfocus/companion-module-kiloview-encoder.git",
+	"bugs": "https://github.com/bitfocus/companion-module-kiloview-encoder/issues",
+	"maintainers": [
+		{
+			"name": "",
```

Comment:
Do you want to maintain this module?

Reply:
i actually copied that file as a template, but not sure who the maintainer was?